doc/source/parameters/dem/insertion_info.rst

```diff
@@ -125,3 +128,16 @@ The List insertion method insert particles at precis coordinates with specific v
     set list z = 0., 3.
 
 * The ``list velocity x``, ``list velocity y``, ``list velocity z``, ``list omega x``, ``list omega y``, ``list omega z`` and ``list diameters`` define the initial translational velocities, the initial angular velocities and diameters of each particles respectively following the same logic as the insertion coordinates.
+
+---------------------
+File
+---------------------
+The "file" insertion method insert particles in a similar way to the "list" insertion method. The main difference between these two methods is the need of an external file provided by the ``particles file`` parameter. This parameter is set at ``particles.input`` by default. This external file has to follow this structure:
```

Comment:
```suggestion
The ``file`` insertion method insert particles in a similar way to the ``list`` insertion method. The main difference between these two methods is the option to use an external file provided by the ``insertion file`` parameter. This parameter is set at ``particles.input`` by default. This file has to follow this structure:
```

Reply:
Also, can you do it for the other methods above (the ``)?

Reply:
Also with actually the file name and not the file (for the parameter name) but we get it